pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -104,6 +104,19 @@ func (k *Kubernetes) CheckUnsupportedKey(komposeObject *kobject.KomposeObject, u
 	return keysFound
 }
 
+// "InitPodSpec creates the pod specification"
```

Comment:
@kadel tests are passing locally when I run `make test`, no idea what is going wrong in Travis.